src/lib/components/multiDropzone/UploadFileCell/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import styles from './style.module.scss';
+import icons from '../icons/index';
+import { UploadStatus, UploadedFile } from '..';
+
+const getUploadingIcon = (type: string) => {
```

Comment:
would be cleaner if you define `type IconType = 'heic' | 'bmp'…` and then the function parameter would be `type: string | IconType`.

Then you can ensure type safety and proper mapping if you add a new icon